www/autocert.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e
+
+function cert {
+    NAME_PREFIX=$1
+    DOMAIN=$2
+    openssl req \
+        -newkey rsa:2048 \
+        -nodes \
+        -keyout $NAME_PREFIX.key \
+        -x509 \
+        -days 1\
+        -out $NAME_PREFIX.cer \
+        -subj "/C=DE/ST=Berlin/L=Berlin/O=deSEC/OU=autocert/CN=$DOMAIN"
+}
+
+cd $1
+pwd
+
+for DOMAIN in \
+    checkipv4.dedyn.${DESECSTACK_DOMAIN} \
+    checkipv6.dedyn.${DESECSTACK_DOMAIN} \
+    checkipv.dedyn.${DESECSTACK_DOMAIN} \
+    dedyn.${DESECSTACK_DOMAIN} \
+    desec.${DESECSTACK_DOMAIN} \
+    update6.dedyn.${DESECSTACK_DOMAIN} \
+    update.dedyn.${DESECSTACK_DOMAIN} \
+    www.dedyn.${DESECSTACK_DOMAIN} \
+    www.desec.${DESECSTACK_DOMAIN} \
+    ; do
+
+    if [ ! -f ${DOMAIN}.cer ] || [ ! -f ${DOMAIN}.key ] ; then
+        rm -f ${DOMAIN}.cer ${DOMAIN}.key
+        cert ${DOMAIN} "${DOMAIN}"
```

Comment:
Why is this a function at all, and why does it have to identical arguments?

Reply:
Dropped function definition and simplified code structure.